app/rbkitConnection.js

```diff
@@ -0,0 +1,48 @@
+import ZMQ from 'zmq';
+import MsgPack from 'msgpack';
+
+const RbkitConnection = (() => {
+  let ip;
+  let requester;
+  let subscriber;
+  let rawProfilingData = [];
```

Comment:
@kgrz This array will be removed when the implementation with redux is ready for other data captures.